packages/platform-test-suite/Dockerfile

```diff
@@ -63,5 +63,3 @@ COPY --from=builder /platform /platform
 RUN cp /platform/packages/platform-test-suite/.env.example /platform/packages/platform-test-suite/.env
 
 EXPOSE 2500 2501 2510
-
-ENTRYPOINT ["/platform/packages/platform-test-suite/bin/test.sh"]
```

Comment:
I believe we added this for purpose so you don't need to pass test.sh command by default. We could change it CMD if you want to use something else from this image